src/libraries/System.Linq/tests/AggregateTests.cs

```diff
@@ -52,7 +59,10 @@ public void SingleElementRunOnce()
             int[] source = { 5 };
             int expected = 5;
 
-            Assert.Equal(expected, source.RunOnce().Aggregate((x, y) => x + y));
+            Assert.All(CreateSources(source), source =>
+            {
+                Assert.Equal(expected, source.RunOnce().Aggregate((x, y) => x + y));
```

Comment:
If the point of `CreateSources` is to test the source data with as many collection types as possible, it probably doesn't make a big difference in tests that use the `RunOnce()` helper since it itself is doing its own wrapping.

Reply:
I just did it for completeness / consistency.